en-GB/lang/numere.nlng

```diff
@@ -121,6 +121,7 @@ PARSERFUNCS_LISTFUNC_FUNC_ELLIPTICD_[NUM]=					ellipticD(x,k)
 PARSERFUNCS_LISTFUNC_FUNC_ELLIPTICE_[NUM]=					ellipticE(x,k)                                              VAL      - Value of the elliptic integral E(x,k) at the position x with 0 <= k < 1. %%DEFINED_REAL%%
 PARSERFUNCS_LISTFUNC_FUNC_ELLIPTICF_[NUM]=					ellipticF(x,k)                                              VAL      - Value of the elliptic integral F(x,k) at the position x with 0 <= k < 1. %%DEFINED_REAL%%
 PARSERFUNCS_LISTFUNC_FUNC_ELLIPTICPI_[NUM]=					ellipticPi(x,n,k)                                           VAL      - Value of the elliptic integral Pi(x,n,k) at the position x with 0 <= k < 1. %%DEFINED_REAL%%
+PARSERFUNCS_LISTFUNC_FUNC_ENDSWITH_[STRING]=				endswith(sStr,sStr)                                         STR      - Gives the value True back, when the second Substring is found at the end of the first string. Otherwise, False.

```

Comment:
```suggestion
PARSERFUNCS_LISTFUNC_FUNC_ENDSWITH_[STRING]=				endswith(sStr,sEnd)                                         LOG      - Returns true, if sStr ends with the string sEnd. Otherwise, false.
```